src/main/java/org/gitlab4j/api/RepositoryFileApi.java

```diff
@@ -43,7 +43,7 @@ public RepositoryFile getFile(String filePath, Integer projectId, String ref) th
         Form form = new Form();
         addFormParam(form, "ref", ref, true);
         Response response = get(Response.Status.OK, form.asMap(),
-                "projects", projectId, "repository", "files", urlEncode(filePath));
+                "projects", projectId, "repository", "files", urlEncodeForPath(filePath));
```

Comment:
Change back to `urlEncode(filePath)` after fix to AbstractApi.urlEncode(String s)